hw3/01-characters.js

```diff
@@ -0,0 +1,58 @@
+// TODO: enforce consistent image size.
+
+// URL for the Thrones API.
+const url = 'https://thronesapi.com/api/v2/Characters';
+const container = document.getElementById('characters');
+
+function buildCharacter(character) {
+  // Create character card.
+  const card = document.createElement('card');
+  card.classList.add('card', 'm-4', 'p-2');
+
+  // Create character image.
+  const img = document.createElement('img');
+  img.src = character.imageUrl;
+  img.classList.add('card-img-top', 'mb-3');
+
+  // Create character name.
+  const name = document.createElement('h2');
+  name.textContent = character.fullName;
+  name.classList.add('card-title', 'fs-5', 'fw-bold', 'text-center');
+
+  // Create character title.
+  const title = document.createElement('p');
+  title.textContent = character.title;
+  title.classList.add('fs-6', 'fw-bold', 'text-center');
+
+  // Add character elements to card.
+  card.appendChild(img);
+  card.appendChild(name);
+  card.appendChild(title);
```

Comment:
Using the regular append() method all three child elements (img, name, and title) could be appended to card at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.

```suggestion
  // Add character elements to card.
  card.append(img, name, title);
```